README.md

```
@@ -374,6 +374,39 @@ tweakcc can automatically round the token counters to the nearest multiple of a
+
+It's neat functionality but the updates occur at what appear to be sporadic intervals. According to [the docs](https://code.claude.com/docs/en/statusline#:~:text=Updates%20run%20at%20most%20every%20300%20ms), _"Updates run at most every 300 ms,"_ but this is inaccurate&mdash;technically, updates are _queued_ for 300 milliseconds, meaning an update is triggered each time the chat history is updated, but is then delayed before execution for 300ms.
+
+In the majority of cases, this behavior is fine. However, if you have a specialized use case, you may need updates to be throttled at 300ms like the documentation states, or even have updates automatically triggered at a specific interval.
+
+tweakcc can patch Claude Code to correct this erratic queuing behavior, making it properly throttle updates at a customizable interval. It can also pace the updating, making it be performed at a regular interval, independent of changes to the chat history.
+
+Here are two demos showing 1) updates triggered every 150ms, and 2) updates triggered by history updates, throttled at 1s. The `update = X` is a custom statusline, where _X_ increments each time the statusline is re-rendered.
+
+| 150ms interval                                     | 1000ms throttling                               |
+| -------------------------------------------------- | ----------------------------------------------- |
+| ![](./assets/statusline_update_interval_150ms.gif) | ![](./assets/statusline_update_throttle_1s.gif) |
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)